vllm/model_executor/models/minicpmv.py

```diff
@@ -960,12 +981,12 @@ def get_vision_hidden_states(
             all_pixel_values.type(dtype),
             patch_attention_mask=patch_attn_mask,
             tgt_sizes=tgt_sizes,
-        ).last_hidden_state
+        )
 
         return self.resampler(vision_embedding, tgt_sizes)
 
     def is_default_weight_loading(self, name: str) -> bool:
-        return "resampler" in name or "vpm" in name
+        return "resampler" in name  #or "vpm" in name
```

Comment:
Is this intended?

Reply:
Yep, this commented code should be deleted

Reply:
So the original code is wrong?

Reply:
similar to :https://github.com/vllm-project/vllm/blob/main/vllm/model_executor/models/minicpmv.py#L868

Reply:
We tested this modified minicpmv26 locally using script  similar to [test_minicpmv](https://github.com/vllm-project/vllm/blob/main/tests/models/decoder_only/vision_language/test_minicpmv.py), and the test passed correctly. For the LoRA test, we trained a LoRA using our internal data, and the inference worked without issues.

Reply:
Actually let me remove the comment by myself